Mathlib/NumberTheory/BernoulliPolynomials.lean

```diff
@@ -149,6 +172,19 @@ theorem bernoulli_eq_sub_sum (n : ℕ) :
   rw [Nat.cast_succ, ← sum_bernoulli n, sum_range_succ, add_sub_cancel_left, choose_succ_self_right,
     Nat.cast_succ]
 
+theorem bernoulli_eq_sub_sum' (n : ℕ) :
```

Comment:
Do we really need all three versions here? The double-prime version doesn't seem to be used in this PR.